views/metas/form/action_button.tmpl

```diff
@@ -0,0 +1,34 @@
+{{ $actionBtn := raw_value_of .ResourceValue .Meta }}
+<div class="qor-field">
+  <button type="button" class="tyk-button tyk-button--md tyk-button--{{ $actionBtn.CssType }}" data-toggle="#modal-{{ .InputId }}"
+    data-target="#modal-{{ .InputId }}">
+    {{ .Label }}
+  </button>
+{{ if $actionBtn.ConfirmModal }}
+  <form action="{{ $actionBtn.URL }}" method="POST">
+    <div class="modal fade" id="modal-{{ .InputId }}" tabindex="-1" role="dialog" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered modal-md" role="document">
+        <div class="modal-content p-2">
+          <div class="modal-header justify-content-center">
+            <h2 class="modal-title">{{ .Label }}</h2>
+          </div>
+          <div class="modal-body">
+          {{ range $actionBtn.ModalContentText }}
+            <p>{{ . }}</p>
+          {{ end }}
+          </div>
+          <div class="modal-footer">
+            <div class="pt-2">
+              <input type="submit" class="btn btn-{{ $actionBtn.CssType }}" value="{{ $actionBtn.ModalConfirmText }}"/>
+              <button type="button" class="btn btn-secondary-outline" data-dismiss="modal">{{ $actionBtn.ModalDismissText }}</button>
+              {{ range $actionBtn.InputHidden }}
+                <input type="hidden" name="{{ .Name }}" value="{{ .Value }}" />
+              {{ end }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </form>
+{{ end }}
+</div>
```

Comment:
It seems you've mixed up some CSS classes, `modal fade`, `p-{number}` and `justify-content-center` for example are Bootstrap ones, which is not used in the admin app.